deploy/helm/listener-operator/values.yaml

```diff
@@ -44,10 +44,28 @@ securityContext:
   # runAsUser: 1000
 
 resources: {}
```

Comment:
```suggestion
controller:
  resources: {}
```

Reply:
If we change this here we should propagate that change to the other operator helm charts as well and it'd be a breaking change everywhere.

Is this mostly cosmetic or are there technical reasons for this change?

Reply:
Mostly cosmetic.

Reply:
In that case I'd rather leave it as is, because so far we have managed to avoid individual PRs per operator for the resources and changing this in the values.yaml would require someone to manually go do it in every repo..

Reply:
True..

Reply:
Thought about it some more, if we rename the fields here, we won't be able to use the macros from the templating to insert default values if these are undefined, so would effectively have to dupliate those ..

Reply:
You can pass a custom context to the `{{ include }}` block, iirc. And we'd still need separate-ish handling for the node agent, I believe?